certified-connectors/PowerToolsLITE/apiDefinition.swagger.json

```diff
@@ -2698,6 +4951,49 @@
         ]
       }
     },
+    "outputDateInfo": {
+      "type": "object",
+      "properties": {
+        "DayOfYear": {
+          "type": "number",
+          "description": "DayOfYear",
+          "x-ms-summary": "DayOfYear"
+        },
+        "DayOfWeek": {
+          "type": "number",
+          "description": "DayOfWeek",
+          "x-ms-summary": "DayOfWeek"
+        },
+        "WeekOfYear": {
+          "type": "number",
+          "description": "WeekOfYear",
+          "x-ms-summary": "WeekOfYear"
+        },
+        "SecondsInDay": {
+          "type": "number",
+          "description": "SecondsInDay",
+          "x-ms-summary": "SecondsInDay"
+        },
+        "MinutesInDay": {
+          "type": "number",
+          "description": "MinutesInDay",
+          "x-ms-summary": "MinutesInDay"
```

Comment:
@eshupps strings in CaMel case are considered non-translatable. Please look through descriptions and x-ms-summary properties. If there's no specific reason to keep it this way, the casing should be fixed. More information about English strings can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
These are properties on a response object which strictly speaking shouldn't be translated but we can change the Description value if you think it would make for a better user experience (I would prefer to leave the x-ms-summary value as is).

Reply:
@eshupps thank you for the explanation!
You can leave the x-ms-summary property as it is in this case. You can change the description for better user experience if you think that it'll be better for users, e.g. `MinutesInDay property for output date information object`. If your users know what this property is needed for, then you can leave it as it is. 

It is recommended to have different descriptions and summaries that don't look like a copy and paste though. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) for more information.